src/client/activation/node/analysisOptions.ts

```diff
@@ -1,18 +1,26 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import { ConfigurationTarget, extensions, WorkspaceConfiguration } from 'vscode';
 import { LanguageClientOptions } from 'vscode-languageclient';
+import * as semver from 'semver';
 import { IWorkspaceService } from '../../common/application/types';
+import { PYLANCE_EXTENSION_ID } from '../../common/constants';
+import { IExperimentService } from '../../common/types';
 
 import { LanguageServerAnalysisOptionsBase } from '../common/analysisOptions';
 import { ILanguageServerOutputChannel } from '../types';
 import { LspNotebooksExperiment } from './lspNotebooksExperiment';
 
+const editorConfigSection = 'editor';
+const formatOnTypeConfigSetting = 'formatOnType';
```

Comment:
```suggestion
const EDITOR_CONFIG_SECTION = 'editor';
const FORMAT_ON_TYPE_CONFIG_SETTING = 'formatOnType';
```
 We use uppercase names for module constants, but other than that this looks good.